pages/hud.coffee

```diff
@@ -95,5 +95,19 @@ handlers =
       " (No matches)"
     countElement.textContent = if showMatchText then countText else ""
 
+  copyToClipboard: (data) ->
+    focusedElement = document.activeElement
+    Clipboard.copy data
+    focusedElement?.focus()
+    window.parent.focus()
+    UIComponentServer.postMessage {name: "unfocusIfFocused", data}
```

Comment:
`unfocusIfFocused`, above, does not take `data`, does it? I'll remove `data` here; let me know if that's incorrect, @mrmr1993.